src/main/java/org/corfudb/protocols/wireprotocol/ManagementBootstrapRequest.java

```diff
@@ -0,0 +1,25 @@
+package org.corfudb.protocols.wireprotocol;
+
+import io.netty.buffer.ByteBuf;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.corfudb.runtime.view.Layout;
+
+/**
+ * Request sent to bootstrap the server with a {@link Layout}.
+ * Created by zlokhandwala on 11/8/16.
+ */
+@Data
+@AllArgsConstructor
+public class ManagementBootstrapRequest implements ICorfuPayload<ManagementBootstrapRequest> {
```

Comment:
This class is unneeded. Can you just pass in CorfuMsg<Layout> to your handlers instead of creating a data type which just contains a layout?